installation/licensing/set-up-your-license.md

```diff
@@ -1,35 +1,86 @@
+
+### Installing a License Key with the VS Extensions
+
+To install your license key by using the [Telerik UI for .NET MAUI Visual Studio extensions]({%slug vs-integration-overview%}):
+
+1. Open Visual Studio.
+1. Go to **Extensions** > **Telerik** **Licensing** > **Download Key**.
+
+    ![.NET MAUI VS Extension License Key](./images/vsx-download-license-key-file.png)
+
+### Installing a License Key with the VS Code Extensions
+
+To install your license key by using the [Telerik UI for .NET MAUI Visual Studio Code extensions]({%slug getting-started-vs-code-integration-overview%}):
+
+1. Open Visual Studio Code and use one of the following approaches:
+* Click the **Telerik Extensions Information** icon in the Visual Studio Code activity bar.
+
+    1. Click on the **Telerik Extensions Information** icon
+    2. Select **MAUI** tab
+    3. Click **Create New Project**.
+    4. The **Telerik UI for .NET MAUI Template Wizard** opens
+
+* From the Visual Studio Code Extensions menu
+
+    1. To open the menu, use:
+        * `Ctrl+Shift+P` on Windows/Linux
+        * `Cmd+Shift+P` on Mac.
+
+    2. Select **Telerik UI for .NET MAUI Template Wizard: Launch** from the menu and press **Enter**. 
+    3. The **Telerik UI for .NET MAUI Template Wizard** opens
+
+2. Check the **TELERIK ACCOUNT DETAILS** field in the wizard and press the **Download License Key File** button.
+
+    ![.NET MAUI VS Extension License Key](./images/telerik-vs-code-extension.png)
+
+### Installing a License Key with the Progress Control Panel
+
+To install your Telerik License Key by using the [Progress Control Panel]({%slug control-panel%}), start the application. It automatically downloads your license key file `telerik-license.txt` to your home directory:
+
+* On Windows `%AppData%\Telerik` or `C:\Users\[windows_username]\%AppData%\Roaming\Telerik`.
+* On Mac/Linux: `~/.telerik/`.
+
+## Manual License Key Installation
+
+To manually download and install a license key for Telerik UI for .NET MAUI:
+
 1. Go to the [License Keys](https://www.telerik.com/account/your-licenses/license-keys) page in your Telerik account.
 
 1. Click the **Download License Key** button.
 
     ![Download a Telerik UI for .NET MAUI License Key](./images/download-license-key.png)
 
-The [Progress Control Panel]({%slug control-panel%}), [automated installers]({%slug automated-installer%}), and the [Visual Studio Extensions]({%slug vs-integration-overview%}) will automatically download and store your license key in your home directory. This makes it available for all projects that you develop on your local machine.
-
-## Activating the Telerik UI for .NET MAUI Components
+### Activating the Telerik UI for .NET MAUI Components
```

Comment:
I think we can make the manual approach a single section for a better flow and less complexity.
```suggestion
```